test/test.jsx

```diff
@@ -874,6 +874,25 @@ test('should render image references', () => {
   )
 })
 
+test('should render footnote with custom options', () => {
+  const input = [
+    'This is a statement[^1] with a citation.',
+    '',
+    '[^1]: This is a footnote for the citation.'
+  ].join('\n')
+
+  assert.equal(
+    asHtml(
+      <Markdown
+        children={input}
+        remarkPlugins={[gfm]}
+        rehypeOptions={{clobberPrefix: 'user-content-'}}
```

Comment:
This value is the default so the test doesn’t test much.
What about passing an empty string or different string instead?